### PR TITLE
ansible-lint: use brew deps

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/a/ansible-lint.rb [hunk 1]
```diff
@@ -43,59 +45,24 @@ class AnsibleLint < Formula
...
-  end
-
-  resource "markdown-it-py" do
-    url "https://files.pythonhosted.org/packages/38/71/3b932df36c1a044d397a1f92d1cf91ee0a503d91e470cbd670aa66b07ed0/markdown-it-py-3.0.0.tar.gz"
-    sha256 "e3f60a94fa066dc52ec76661e37c851cb232d92f9886b15cb560aaada2df8feb"
-  end
-
-  resource "mdurl" do
-    url "https://files.pythonhosted.org/packages/d6/54/cfe61301667036ec958cb99bd3efefba235e65cdeb9c84d24a8293ba1d90/mdurl-0.1.2.tar.gz"
-    sha256 "bb413d29f5eea38f31dd4754dd7377d4465116fb207585f97bf925588687c1ba"
```
Comment (chenrui333) [inline Formula/a/ansible-lint.rb:78]: why did these get removed

Comment (branchvincent) [reply Formula/a/ansible-lint.rb:78]: `rich` (should) provide those:

```console
$ pipgrip --tree rich
rich (13.7.0)
├── markdown-it-py>=2.2.0 (3.0.0)
│   └── mdurl~=0.1 (0.1.2)
└── pygments<3.0.0,>=2.13.0 (2.17.2)
```

Comment (chenrui333) [reply Formula/a/ansible-lint.rb:78]: I see, for now, I just add them back.

**Comment by chenrui333** [comment]
close for now.

**Pull request closed without merging**